pdfminer/ccitt.py

```diff
@@ -454,8 +452,9 @@ def _flush_line(self):
         return
 
     def _do_vertical(self, dx):
-        #print '* vertical(%d): curpos=%r, color=%r' % (dx, self._curpos, self._color)
-        #print '  refline:', self._get_refline(self._curpos+1)
+        # print '* vertical(%d): curpos=%r, color=%r' % (dx, self._curpos,
+        # self._color)
+        # print '  refline:', self._get_refline(self._curpos+1)
```

Comment:
These sort of comments should be removed or replaced by `logger.debug()` statements.

Reply:
I've changed my mind (really quick ;). Let's keep the comments for now (in this PR) and solely focus on code-style enforcement.